pkgs/by-name/pu/purescm/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildNpmPackage
+, fetchNpmDeps
+, testers
+}:
+
+let
+  package = buildNpmPackage rec {
+    pname = "purescm";
+    version = "1.0.0";
+
+    src = ./.;
+
+    dontNpmBuild = true;
+
+    npmDeps = fetchNpmDeps {
+      inherit src;
+      hash = "sha256-rSm40Q9A5xfMQIsH6QQaCcemU9LnLmvGdLmII4nApsI=";
+      postPatch = ''
+        cp ${./package-lock.json} package-lock.json
+      '';
+    };
+
+    installPhase = ''
+      mkdir -p $out/share/${pname}
+      cp -r node_modules/ $out/share/${pname}
+      mkdir -p $out/bin
+      ln -s $out/share/${pname}/node_modules/.bin/purescm $out/bin
+
```

```suggestion
      mkdir -p $out/share/${pname}
      cp -r node_modules/ $out/share/${pname}
      mkdir -p $out/bin
      ln -s $out/share/${pname}/node_modules/.bin/purescm $out/bin
```
Please don't substitute pname wherever can be.